app.test.js

```diff
@@ -70,14 +70,104 @@ describe('getReviews', () => {
           })
         });
     });
-    test("Responds with review objects ordered by date in descending", () => {
+    test("Responds with an array of review objects where category = euro game", () => {
+      return request(app)
+        .get("/api/reviews?category=dexterity")
```

Comment:
test description will need to be updated to reflect the test of category=dexterity